access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt

```diff
@@ -226,6 +227,18 @@ class AccessMemberCardFragment : BaseViewModelFragment<AccessMemberCardViewModel
                     requireActivity().hideSoftKeyboard()
                 }.disposedBy(disposeBag)
 
+        zipCode.setOnEditorActionListener { _, actionId, _ ->
+            when (actionId) {
+                EditorInfo.IME_ACTION_DONE -> {
+                    val isFormFilled = !zipCode.text.isNullOrBlank() && !memberId.text.isNullOrBlank()
+                    if (isFormFilled) {
+                        viewModel.onSignInClick()
+                    }
+                    true
+                }
+                else -> false
```

Comment:
I suppose in the future we should consider adding an error message or something to highlight zip and/or member id if they're empty.

Reply:
Currently, sign in button is disabled if the form is not filled.